app/code/Magento/Pwa/Test/Mftf/Data/FastlyImageResizeData.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<entities xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="urn:magento:mftf:DataGenerator/etc/dataProfileSchema.xsd">
+
+    <entity name="FastlyImageResize" type="customer">
```

Comment:
Will apply to entire review; we can make this test general, since image optimization is built into upward-js, and this test should still pass.

Reply:
Done.